hydroengine_service/main.py

```diff
@@ -1498,6 +1498,28 @@ def get_gebco_data():
         mimetype='application/json'
     )
 
+@v1.route('/get_elevation_data', methods=['GET', 'POST'])
+@flask_cors.cross_origin()
+def get_elevation_data():
+    r = request.get_json()
+    datasets = r.get('datasets', None)
+    image_id = r.get('imageId', None)
+    source = None
+    if datasets:
+        source = datasets
+    if image_id:
+        source = image_id
+
+    image_info = dgds_functions.generate_elevation_map(source)
+    if not image_info:
+        raise error_handler.InvalidUsage('No images returned.')
```

Comment:
if error catched, response not fired with message

Reply:
removed, check for errors in the function generate_elevation_map itself.